packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/grafana/dashboards-1.14/alertmanager-overview.yaml.patch

```diff
@@ -13,11 +13,603 @@
 -{{- end }}
 \ No newline at end of file
++    {
++        "__inputs": [
++
```

Comment:
Does the `\ No newline at end of file` interfere with the subsequent definition here? I think this may apply to all dashboards (I did not verify each one in detail)

Reply:
Should be good here, just part of the diff forma: https://www.gnu.org/software/diffutils/manual/html_node/Incomplete-Lines.html